src/NUnitEngine/nunit.engine.tests/Runners/TestEngineRunnerTests.cs

```diff
@@ -39,7 +39,7 @@ namespace NUnit.Engine.Runners.Tests
     //[TestFixture(typeof(ProcessRunner))]
     [TestFixture(typeof(MultipleTestDomainRunner), 1)]
     [TestFixture(typeof(MultipleTestDomainRunner), 3)]
-    [TestFixture(typeof(MultipleTestProcessRunner), 1)]
+    //[TestFixture(typeof(MultipleTestProcessRunner), 1)]
```

Comment:
@CharliePoole - could you create this issue? I feel you have a better idea of the background here than I do!